libs/shared/src/lib/functions/html-decode.ts

```diff
@@ -0,0 +1,4 @@
+export function htmlDecode(input: string): string {
+    var doc = new DOMParser().parseFromString(input, "text/html");
+    return doc.documentElement.textContent;
+}
```

Comment:
Method used to handle \&amp; -> & outside of HTML files

Reply:
oh good, hopefully we can stop having amps in places where people don't want them. 👍